strapi/strapi
# Auto close the edit modal on form submit

### What does it do?

~It resets the reducer state so the edit modal can disappear once the form is submitted.~
Closes the modal after submitting
Added a new test to make sure it closes every time the form is submitted
~Also adjusted an existing test.~

### Why is it needed?

The edit modal is not auto closing and it gives the impression of not working

### How to test it?

1. Open any entity table, eg users
2. Click on the cog AKA configure view
3. Edit any label
4. Click on Finish
5. Modal should now close

### Related issue(s)/PR(s)

Closes #16347
